downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -66,8 +66,146 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ### New features
 
+#### Support to provide values for configurable variables through environment variables
+
+Configurable values can now be provided through environment variables using the following syntax.
+
+```
+BAL_CONFIG_VAR_key=value
+```
+
+The key conforms to the structure `ORG_MODULE_VARIABLE`, where each part in the structured identifier is converted to uppercase, and dots are converted to underscores.
+
+The environment variable-based configuration is supported for configurable variables of `boolean`, `int`, `float`, `decimal`, `string`, and `xml` types.
+
+For example, if the configurable variable is defined in the following way,
+
+```ballerina
+configurable int port = ?;
+```
+
+The values can be provided through environment variables as follows.
+
+If the configurable variable is defined in the default module or if a single Ballerina file is being used:
+
+For Windows:
+```
+$ set BAL_CONFIG_VAR_PORT=9090
+```
+
+For Linux/macOS:
+```
+$ export BAL_CONFIG_VAR_PORT=9090
+```
+
+If the configurable variable is defined in a different module with name `foo.bar` from the same organization:
+
+For Windows:
+```
+$ set BAL_CONFIG_VAR_FOO_BAR_PORT=9090
+```
+
+For Linux/macOS:
+```
+$ export BAL_CONFIG_VAR_FOO_BAR_PORT=9090
+```
+
+If the configurable variable is defined in a module with name `foo.bar` from a different organization called `testOrg`.
+
+For Windows:
+```
+$ set BAL_CONFIG_VAR_TESTORG_FOO_BAR_PORT=9090
+```
+
+For Linux/macOS:
+```
+$ export BAL_CONFIG_VAR_TESTORG_FOO_BAR_PORT=9090
+```
```

Comment:
Shouldn't we add a note on what takes priority if specified in multiple ways (Config file, command argument, etc.)